spyder/plugins/remoteclient/plugin.py

```diff
@@ -213,16 +213,39 @@ def load_conf(self, config_id):
         if not options:
             return {}
 
-        if options["client_keys"]:
+        if options["config"]:
+            # Only `host` and list with path to config file as `config`
+            # are required
+            options["config"] = [options["config"]]
+
+            # Optional configuration vslues mapping (`port`, `username`,
+            # `client_keys`, `passphrase` and `password`)
+            if options["port"] == 0:
+                # If 0 is set (`From file` option) ignore value
```

Comment:
```suggestion
                # Ignore value if 0 is set because it means the port we'll be
                # read from the config file.
```

Improve comment according to my suggestions below.